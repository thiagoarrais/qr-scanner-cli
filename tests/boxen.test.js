const { greenBox } = require('../src/infrastructure/boxen')

test('Should fail', async () => {
  expect(greenBox('sample input')).toMatchSnapshot()
})
