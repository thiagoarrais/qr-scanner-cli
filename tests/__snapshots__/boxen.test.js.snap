// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should fail 1`] = `
"[32m╔══════════════════╗[39m
[32m║[39m                  [32m║[39m
[32m║[39m   sample input   [32m║[39m
[32m║[39m                  [32m║[39m
[32m╚══════════════════╝[39m"
`;
